testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/remote/AbstractHotRodWebFailoverTestCase.java

```diff
@@ -47,4 +67,44 @@ public void beforeTestMethod() {
         NodeUtil.start(this.controller, this.nodes);
         NodeUtil.deploy(this.deployer, this.deployments);
     }
+
+    @Override
+    protected void testGracefulSimpleFailover(URL baseURL1, URL baseURL2, URL baseURL3) throws Exception {
+        super.testGracefulSimpleFailover(baseURL1, baseURL2, baseURL3);
+
+        String readResource = String.format("/subsystem=infinispan/remote-cache-container=web/remote-cache=%s:read-resource(include-runtime=true, recursive=true)", this.deploymentName);
+        String resetStatistics = String.format("/subsystem=infinispan/remote-cache-container=web/remote-cache=%s:reset-statistics", this.deploymentName);
+
+        ModelNode result = execute(this.client1, readResource);
+        Assert.assertNotEquals(0L, result.get("hits").asLong());
+        Assert.assertNotEquals(0L, result.get("writes").asLong());
+        Assert.assertNotEquals(0L, result.get("near-cache-size").asLong());
+
+        result = execute(this.client2, readResource);
+        Assert.assertEquals(0L, result.get("hits").asLong());
+        Assert.assertEquals(0L, result.get("writes").asLong());
+        Assert.assertEquals(0L, result.get("near-cache-size").asLong());
+
+        result = execute(this.client3, readResource);
+        Assert.assertNotEquals(0L, result.get("hits").asLong());
+        Assert.assertNotEquals(0L, result.get("writes").asLong());
+        Assert.assertNotEquals(0L, result.get("near-cache-size").asLong());
+
+        execute(this.client1, resetStatistics);
+        execute(this.client2, resetStatistics);
+        execute(this.client3, resetStatistics);
+
+        // These metrics should have reset
+        result = execute(this.client1, readResource);
+        Assert.assertEquals(0L, result.get("hits").asLong());
+        Assert.assertEquals(0L, result.get("writes").asLong());
+
+        result = execute(this.client2, readResource);
+        Assert.assertEquals(0L, result.get("hits").asLong());
+        Assert.assertEquals(0L, result.get("writes").asLong());
+
+        result = execute(this.client3, readResource);
+        Assert.assertEquals(0L, result.get("hits").asLong());
+        Assert.assertEquals(0L, result.get("writes").asLong());
+    }
```

Comment:
@pferraro test sequence is:

response = client.execute(new HttpGet(uri1));
response = client.execute(new HttpGet(uri2));
response = client.execute(new HttpGet(uri3));

lifecycle.stop(NODE_1);

response = client.execute(new HttpGet(uri2));
response = client.execute(new HttpGet(uri2));
response = client.execute(new HttpGet(uri3));
response = client.execute(new HttpGet(uri3));

lifecycle.start(NODE_1);

response = client.execute(new HttpGet(uri2));
response = client.execute(new HttpGet(uri2));
response = client.execute(new HttpGet(uri3));

lifecycle.stop(NODE_2);

response = client.execute(new HttpGet(uri1));
response = client.execute(new HttpGet(uri1));
response = client.execute(new HttpGet(uri3));
response = client.execute(new HttpGet(uri3));

lifecycle.start(NODE_2);

response = client.execute(new HttpGet(uri1));
response = client.execute(new HttpGet(uri1));

hence, NODE_2 never gets hit after restart: that's the reason why it's statistics remain at 0, correct?